smartsim/database/orchestrator.py

```diff
@@ -35,8 +35,6 @@
 from shlex import split as sh_split
 
 import psutil
-from smartredis import Client, ConfigOptions
-from smartredis.error import RedisReplyError
```

Comment:
Ditto "shouldn't matter, but symbols are no longer defined and being used" comment here. Consider setting up some mocks or removing the usage to get past the CI.

Reply:
added mocks